turbo/cli/default_flags.go

```diff
@@ -175,6 +175,9 @@ var DefaultFlags = []cli.Flag{
 	&utils.BeaconApiProtocolFlag,
 	&utils.BeaconApiIdleTimeoutFlag,
 
+	&utils.CaplinBackfillingFlag,
```

Comment:
i think these flags should be added to https://github.com/ledgerwatch/erigon/blob/devel/cmd/caplin/caplinflags/flags.go